copyright.js

```diff
@@ -1,5 +1,5 @@
 /**
- * © <%= YEAR %> Liferay, Inc. <https://liferay.com>
- *
+ * SPDX-FileCopyrightText: © <%= YEAR %> Liferay, Inc. <https://liferay.com>
  * SPDX-License-Identifier: BSD-3-Clause
  */
+
```

Comment:
Blank line is intentional here; ensures that we always have a separation between header and the rest of the source (before this PR, there were a handful of files — about half a dozen — that didn't have this).